src/main/java/core/basesyntax/strategy/StrategyCalculator.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+
+public interface StrategyCalculator {
```

Comment:
Better to rename this interface to "OperationHandler" and rename the method to "handle"
Change names for all implementation to according to the name convention